<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AzkarDialog</class>
 <widget class="QDialog" name="AzkarDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>345</width>
    <height>139</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>اعدادت الأذكار</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <property name="locale">
   <locale language="Arabic" country="SaudiArabia"/>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>اعدادات الأذكار</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="azkarEnableCheckBox">
        <property name="text">
         <string>تشغيل الاذكار ، كل :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QSpinBox" name="minuteSpinBox">
        <property name="suffix">
         <string>  دقيقة</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>210</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="automaticCloseCheckBox">
        <property name="text">
         <string>غلق تلقائي ، بعد:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QSpinBox" name="secondSpinBox">
        <property name="suffix">
         <string>  ثانية</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>9999</number>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>89</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="okButton">
     <property name="text">
      <string>موافق</string>
     </property>
     <property name="icon">
      <iconset resource="gui.qrc">
       <normaloff>:/images/ok.png</normaloff>:/images/ok.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>الغاء</string>
     </property>
     <property name="icon">
      <iconset resource="gui.qrc">
       <normaloff>:/images/cancel.png</normaloff>:/images/cancel.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>348</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>azkarEnableCheckBox</tabstop>
  <tabstop>minuteSpinBox</tabstop>
  <tabstop>automaticCloseCheckBox</tabstop>
  <tabstop>secondSpinBox</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources>
  <include location="gui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>AzkarDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>15</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>AzkarDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>281</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>88</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
